Gemfile

```diff
@@ -233,7 +233,12 @@ def override_gem(name, *args)
 
     calling_file = caller_locations.detect { |loc| !loc.path.include?("lib/bundler") }.path
     gem(name, *args).tap do
-      warn "** override_gem: #{name}, #{args.inspect}, caller: #{calling_file}" unless ENV["RAILS_ENV"] == "production"
+      verbose = if ENV["GEMFILE_SILENT"]
+                  %w(false 0).include?(ENV["GEMFILE_SILENT"])
```

Comment:
> Also supporting GEMFILE_SILENT=0 or GEMFILE_SILENT=false to force seeing them in production.

(And I don't understand this part, either - regardless of the truthiness (false or true) you can force in production...)